release/release-notes/data-prepper.release-notes-1.3.0.md

```diff
@@ -0,0 +1,58 @@
+### Bug Fixes
+* Allow stdout and file sink to output generic object type ([#1192](https://github.com/opensearch-project/data-prepper/pull/1192))
+* Fixed issue where Spring was unable to find the PrometheusMeterRegistry Bean ([#1019](https://github.com/opensearch-project/data-prepper/pull/1019))
+* Use BlockingTaskExecutor in OtelTraceSource ([#745](https://github.com/opensearch-project/data-prepper/pull/745))
+
+### Infrastructure
+* Upload Maven artifacts as part of Release build ([#1181](https://github.com/opensearch-project/data-prepper/pull/1181))
+* Updated the release build to push the Docker image and upload archives ([#1151](https://github.com/opensearch-project/data-prepper/pull/1151))
+* Update Gradle project to produce only tar.gz archives ([#1132](https://github.com/opensearch-project/data-prepper/pull/1132))
+* Add simple integration tests for AggregateProcessor ([#1046](https://github.com/opensearch-project/data-prepper/pull/1046))
+* Assemble the data-prepper-core uber-jar using Zip64 ([#820](https://github.com/opensearch-project/data-prepper/pull/820))
+
+### Documentation
+* Example on log ingestion from Kubernetes containers ([#729](https://github.com/opensearch-project/data-prepper/pull/729))
+* Update copyright headers in release scripts ([#933](https://github.com/opensearch-project/data-prepper/pull/933))
+* Update copyright headers in data-prepper subprojects ([#776](https://github.com/opensearch-project/data-prepper/pull/776), [#928](https://github.com/opensearch-project/data-prepper/pull/928))
+* Example on Data-Prepper ECS Firelens integration ([#704](https://github.com/opensearch-project/data-prepper/pull/704))
+
+### Maintenance
+* Delete outdated Kibana trace analytics example ([#1135](https://github.com/opensearch-project/data-prepper/pull/1135))
+* Update Open Distro usages to OpenSearch in scripts ([#1086](https://github.com/opensearch-project/data-prepper/pull/1086))
+* Upgrade docker-compose.yml files from ODFE to OpenSearch ([#847](https://github.com/opensearch-project/data-prepper/pull/847))
+* Update the ADOT example to OpenSearch ([#703](https://github.com/opensearch-project/data-prepper/pull/703))
+* Migrate demo sample application to use opensearch and opensearch dashboards instead of ODFE and kibana ([#666](https://github.com/opensearch-project/data-prepper/pull/666))
+* Update log ingestion example to use latest data-prepper docker image ([#752](https://github.com/opensearch-project/data-prepper/pull/752))
+
+### Refactoring
+* Deprecating Record and RecordMetaData ([#954](https://github.com/opensearch-project/data-prepper/pull/954))
+* FileSink to use Event model ([#750](https://github.com/opensearch-project/data-prepper/pull/750))
+* StringPrepper to use Event model ([#753](https://github.com/opensearch-project/data-prepper/pull/753))
+* Rename Prepper to Processor ([#655](https://github.com/opensearch-project/data-prepper/pull/655), [#667](https://github.com/opensearch-project/data-prepper/pull/667))
```

Comment:
I'd like to make a call-to-action somewhere in the release notes. I'd like to let users know that along with this change, we have updated the Pipeline configuration to support `processor:` and that we will be removing `prepper:` in 2.0. I think clearly noting this deprecation is important.

I'm open to different ways of doing it, but maybe have a section higher up for deprecations?

```
### Deprecations
```

Reply:
I will update the description and move it to a Deprecation section